src/ohri-form.scss

```diff
@@ -40,12 +40,15 @@
 }
 
 .minifiedFormContentBody {
-  margin-bottom: 0px !important;
+  margin-bottom: 3rem !important;
 }
 
 .minifiedButtons {
   width: 100%;
   height: 63.5px;
+  position: absolute ;
+  bottom: 0;
+  margin-top: 10rem;
```

Comment:
We can try remove the `margin-bottom` i think it would not have any effect on the UI

Reply:
Nice observation, fixed. Thanks